src/main/java/org/example/commands/DeleteDependencyCommand.java

```diff
@@ -16,6 +16,11 @@ public void execute(Map<String, Node> nodeDependencies) {
     String parentId = scanner.nextLine();
     String childId = scanner.nextLine();
 
+    if (parentId.equals("") || childId.equals("") || childId.equals(null) || parentId.equals(
```

Comment:
always check null first then empty.

Reply:
childId.equals(null) -> if childId is actually null then there will null pointer exception

Reply:
changed the order